sdk/openai/Azure.AI.OpenAI/tests/AudioTranscriptionsTests.cs

```diff
@@ -0,0 +1,152 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.IO;
+using System.Threading.Tasks;
+using Azure.Core.TestFramework;
+using NUnit.Framework;
+
+namespace Azure.AI.OpenAI.Tests;
+
+public class AudioTranscriptionsTest : OpenAITestBase
```

Comment:
nit: It should be called AudioTranscriptionsTests (plural rather than singular). Also, this class also includes tests for translations, so it might be good to rename it or separate it into two files accordingly.

Reply:
Very fair nitpick! I did the full treatment:

- Renamed to `AudioTranscriptionTests` (both plural for tests and not-plural for the scenario, c.f. completions) 
- Split out separate `AudioTranslationTests` to match